feat/v1.5.1 (#217)
smartmind-team/thanosql-docs: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_table_templates.md

    ```shell
      curl -X 'GET' \
      'https://{your-engine-url}/api/v1/table_template/?search={search}&order_by={order_by}&latest={latest}' \
      -H 'accept: application/json' \
      -H 'Authorization: Bearer Issued_API_TOKEN'
    ```

### __Parameters__

> - `search`: Word(s) that the names of queried table templates must contain (defaults to empty string).

Reviewer: ```suggestion
- `search`: Word(s) that the table templates' name must contain (defaults to empty string).
```

- `order_by`: How the query results should be ordered, there are three possible values: 'recent', based on date of creation from latest to oldest, 'name_asc', based on name from A to Z, and 'name_desc', based on name from Z to A (defaults to 'recent').
- `latest`: A boolean value that can be set to `True` if you only want to return the latest version of each table template. Otherwise, it returns all available versions (default: `False`).

## **`GET` /table_template/{table_template_name}**

> Use this method to get the table template object of a certain name. If needed, a specific or latest version can also be added to the request.

Reviewer: ```suggestion
Use this method to get the table template object with the given name. If needed, a specific or latest version can also be added to the request.
```

    ```shell
      curl -X 'GET' \
      'https://{your-engine-url}/api/v1/table_template/{table_template_name}?version={version}' \
      -H 'accept: application/json' \
      -H 'Authorization: Bearer Issued_API_TOKEN'
    ```

### __Response__

> This API's response has the following structure:

Reviewer: ```suggestion
The API response has the following structure:
```

        "compatibility": "string"
        }'
    ```

## **`DELETE` /table_template/{table_template_name}**

> Use this method to delete table templates of a certain name. If needed, it is also possible to delete only a specific version.

Reviewer: ```suggestion
Use this method to delete table templates with the given name. If needed, it is also possible to delete only a specific version.
```
